draft-ietf-oauth-sd-jwt-vc.md

```diff
@@ -1570,6 +1567,10 @@ for their contributions (some of which substantial) to this draft and to the ini
 
 # Document History
 
+-10
+
+* Editorial changes
+
```

Comment:
we're not on -10 yet https://datatracker.ietf.org/doc/draft-ietf-oauth-sd-jwt-vc/08/ 
```suggestion
-10

* Editorial changes

```

Reply:
What does this change? except for the empty line?

Reply:
```suggestion
```